core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -82,6 +102,28 @@ angular.module('oppia').controller('TranslationModalController', [
       });
 
     $scope.onContentClick = function($event) {
+      const paragraphCopyValidation = $scope.validateParagraphCopy($event);
+      if (paragraphCopyValidation) {
+        return $scope.triedToCopyText = true;
+      }
+      $scope.triedToCopyText = false;
+      return $scope.copyContent($event);
+    };
+
+    $scope.validateParagraphCopy = function($event) {
+      // Mathematical equations are also wrapped by <p> elements.
+      // Hence, math elements should be allowed to be copied.
+      // It is related to issue 11683.
```

Comment:
```suggestion
      // See issue #11683.
```

"It" is a bit ambiguous here.

Reply:
Done